### PR TITLE
Added background, buttons and textview. Applied it to the login screen.

### PR DESCRIPTION
**Pull request by RSchipper**

### CODE REVIEW

File: app/src/main/java/android/spirithunt/win/MainActivity.java [hunk 1]
```diff
@@ -33,7 +33,7 @@ protected void onCreate(Bundle savedInstanceState) {
         SharedPreferences sharedPref = this.getSharedPreferences(getString(R.string.preferences_file), Context.MODE_PRIVATE);
 
         if(sharedPref.contains(getString(R.string.saved_jwt))) {
-            this.showMenu();
+            this.showLogin();
```
Comment (roelofr) [inline app/src/main/java/android/spirithunt/win/MainActivity.java]: Remove the `showMenu` method too :wink:

File: app/src/main/java/android/spirithunt/win/controller/LoginController.java [hunk 2]
```diff
@@ -79,11 +79,16 @@ protected void onCreate(Bundle savedInstanceState) {
         setContentView(R.layout.login_view);
     }
 
+    public void signUp(View view) {
+        //TODO: something useful
+        System.out.println("Open the sign-up view or something useful");
```
Comment (roelofr) [inline app/src/main/java/android/spirithunt/win/controller/LoginController.java]: `System.out.println` doesn't show to users, maybe throw a NotImplementedException instead?

File: app/src/main/java/android/spirithunt/win/gui/CustomEditText.java [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+import android.spirithunt.win.provider.FontProvider;
+import android.support.v7.widget.AppCompatEditText;
+import android.util.AttributeSet;
+
+/**
+ * @author Remco Schipper
+ * @see <a href="http://stackoverflow.com/a/16648457">How to set a particular font for a button text in android?</a>
+ */
+
+public class CustomEditText extends AppCompatEditText {
```
Comment (roelofr) [inline app/src/main/java/android/spirithunt/win/gui/CustomEditText.java]: Missing description in Javadoc

File: app/src/main/java/android/spirithunt/win/gui/CustomTextView.java [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+import android.spirithunt.win.provider.FontProvider;
+import android.support.v7.widget.AppCompatEditText;
+import android.util.AttributeSet;
+
+/**
+ * @author Remco Schipper
+ * @see <a href="http://stackoverflow.com/a/16648457">How to set a particular font for a button text in android?</a>
+ */
+
+public class CustomTextView extends AppCompatEditText {
```
Comment (roelofr) [inline app/src/main/java/android/spirithunt/win/gui/CustomTextView.java]: Missing description in Javadoc

File: app/src/main/res/drawable/btn_normal_main_style.xml [hunk 5]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android" >
+    <item android:drawable="@drawable/btn_normal_main" />
+    <item>
+        <selector xmlns:android="http://schemas.android.com/apk/res/android" >
+            <item android:state_pressed="true" >
+                <shape android:shape="rectangle"  >
+                    <corners android:radius="12dip" />
+                    <gradient android:angle="90" android:startColor="#880f0f10" android:centerColor="#880d0d0f" android:endColor="#885d5d5e"/>
```
Comment (roelofr) [inline app/src/main/res/drawable/btn_normal_main_style.xml]: Variables?

File: app/src/main/res/drawable/btn_normal_blue_style.xml [hunk 6]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android" >
+    <item android:drawable="@drawable/btn_normal_blue" />
+    <item>
+        <selector xmlns:android="http://schemas.android.com/apk/res/android" >
+            <item android:state_pressed="true" >
+                <shape android:shape="rectangle"  >
+                    <corners android:radius="12dip" />
+                    <gradient android:angle="90" android:startColor="#880f0f10" android:centerColor="#880d0d0f" android:endColor="#885d5d5e"/>
```
Comment (roelofr) [inline app/src/main/res/drawable/btn_normal_blue_style.xml]: Variables?

File: app/src/main/res/drawable/btn_normal_red_style.xml [hunk 7]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android" >
+    <item android:drawable="@drawable/btn_normal_red" />
+    <item>
+        <selector xmlns:android="http://schemas.android.com/apk/res/android" >
+            <item android:state_pressed="true" >
+                <shape android:shape="rectangle"  >
+                    <corners android:radius="12dip" />
+                    <gradient android:angle="90" android:startColor="#880f0f10" android:centerColor="#880d0d0f" android:endColor="#885d5d5e"/>
```
Comment (roelofr) [inline app/src/main/res/drawable/btn_normal_red_style.xml]: Variables?

File: app/src/main/res/drawable/edittext_normal_main.xml [hunk 8]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android" android:shape="rectangle">
+    <solid android:color="#33FFFFFF"/>
```
Comment (roelofr) [inline app/src/main/res/drawable/edittext_normal_main.xml]: Variables?

File: app/src/main/res/drawable/textview_normal_main.xml [hunk 9]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android" android:shape="rectangle">
+    <stroke android:width="0dp" android:color="#BFFFFFFF" />
```
Comment (roelofr) [inline app/src/main/res/drawable/textview_normal_main.xml]: Variables?

File: app/src/main/res/layout/login_view.xml [hunk 10]
```diff
@@ -1,33 +1,62 @@
...
+            android:layout_marginBottom="7dp" />
+
+        <android.spirithunt.win.gui.CustomEditText style="@style/editText_normal_main"
+            android:id="@+id/password"
+            android:layout_width="fill_parent"
+            android:layout_height="wrap_content"
+            android:hint="@string/password_hint"
+            android:layout_below="@id/email"
+            android:inputType="textPassword"
+            android:layout_marginBottom="7dp" />
```
Comment (roelofr) [inline app/src/main/res/layout/login_view.xml]: Variable

File: app/src/main/res/layout/login_view.xml [hunk 11]
```diff
@@ -1,33 +1,62 @@
...
+
+        <android.spirithunt.win.gui.CustomButton
+            style="@style/btn_normal_main_large"
+            android:id="@+id/login_button"
+            android:text="@string/btn_login"
+            android:onClick="submit"
+            android:layout_marginBottom="25dp" />
```
Comment (roelofr) [inline app/src/main/res/layout/login_view.xml]: Variable

File: app/src/main/res/layout/login_view.xml (hunk 10 above)
Comment (RSchipper) [inline app/src/main/res/layout/login_view.xml]: Seems kinda pointless to me... The variable is only usable for that screen.

File: app/src/main/res/drawable/textview_normal_main.xml (hunk 9 above)
Comment (RSchipper) [inline app/src/main/res/drawable/textview_normal_main.xml]: Seems kinda pointless to me... The variable is only usable for the textview.

File: app/src/main/res/drawable/edittext_normal_main.xml (hunk 8 above)
Comment (RSchipper) [inline app/src/main/res/drawable/edittext_normal_main.xml]: Seems kinda pointless to me... The variable is only usable for the edittext.

**Pull request merged**